### PR TITLE
 user authentication system

### PR DESCRIPTION
**Pull request by mb556**
created a file called auth.js for authentication purposes

relates #2

### CODE REVIEW

File: microblog-backend/.env
Comment (PL1106) [inline microblog-backend/.env:1]: change the port from 5000 to 3001

Comment (PL1106) [reply microblog-backend/.env:1]: You need a git ignore and remove all the unwated files from the PR because it has too many files.

Comment (mb556) [reply microblog-backend/.env:1]: read the file before u comment, where does the port say 5000, review it properly.

Comment (camelPhonso) [reply microblog-backend/.env:1]: Screenshot 2024-09-05 at 14 25 02

Right here my dude - this file should also be in a `.gitignore`

**Pull request merged**